elife/nginx-php7.sls

```diff
@@ -32,6 +40,27 @@ php-fpm:
 
     service.running:
         - name: php7.0-fpm
+        - enable: True
         - require:
             - file: php-fpm
-            - file: php-fpm-config            
+            - file: php-fpm-config
+            - php-fpm-pool
+        - watch:
+            - file: php-fpm-config
+            - file: php-fpm-pool
+
+php-cachetool:
+    file.managed:
+        - name: /usr/local/bin/cachetool
+        - source: https://gordalina.github.io/cachetool/downloads/cachetool.phar
+        - source_hash: md5=fa7ce33b37dba2642329b9a6bdc720b1
```

Comment:
Looks like this will break on every release, since there's no version in the URI...

Reply:
It defines a `bin` folder so a `composer global require ...` could be more helpful in managing its version? https://github.com/gordalina/cachetool/blob/master/composer.json#L35
It would probably be slower than a phar and an MD5 check.

Reply:
Or https://github.com/gordalina/cachetool/releases to get a tarball.

Reply:
Don't hugely want to get involved in a more complex build process for a little tool. Could open an issue upstream and place the PHAR on S3 for now?

Reply:
Global Composer dependencies aren't great, due to constraint conflicts.

Reply:
I suggest we have this deployed first, and use it to try and optimise values (by looking at what's actually happening).

Reply:
:+1: for as-is and .phar on S3 for the medium-term, may not even prove useful.

Reply:
Moved to #85.